conda_recipe_manager/parser/recipe_parser_convert.py

```diff
@@ -447,6 +447,10 @@ def render_to_new_recipe_format(self) -> tuple[str, MessageTable]:
           - https://github.com/conda-incubator/ceps/blob/main/cep-14.md
 
         (As of writing there is no official name other than "the new recipe format")
+        :returns: Returns a tuple containing:
+            - The converted recipe, as a string
+            - A MessageTbl instance that contains error logging
```

Comment:
```suggestion
            - A `MessageTbl` instance that contains error logging
```

Just a little nitpick here for consistency's sake